EZSwiftExtensionsTests/DoubleTests.swift

```diff
@@ -42,6 +42,12 @@ extension DoubleTests {
         XCTAssertNotEqual(Double(anotherDouble.toInt), anotherDouble)
     }
     
+    func testCGFloat() {
+        XCTAssertEqual(double.toCGFloat, 3.14)
+        XCTAssertEqual(otherDouble.toCGFloat, -147.9564)
+        XCTAssertEqual(anotherDouble.toCGFloat, 231349.678450342834857392948575)
```

Comment:
This value is obtained from the present test cases. Ill change it to something that is more robust.